### PR TITLE
first results on properties of p

### PR DESCRIPTION
**Pull request by dpo**
@MohamedLaghdafHABIBOULLAH Please check this carefully.

### CODE REVIEW

**Review by MohamedLaghdafHABIBOULLAH — APPROVED** [review]

File: paper/rn2.tex [hunk 1]
```diff
@@ -157,29 +158,139 @@ \section{Background}%
...
+  Therefore, \(0 \in \partial m(0; x, \sigma) = \nabla \varphi(0; x) + \partial \omega(0; x)\), where \(\omega(s; x) := \psi(s; x) + \chi(s \mid [\ell - x, u - x])\).
+  Because the constraint qualification is satisfied at \(s = 0\), \(\partial \omega(0; x) = \partial \psi(0; x) + N(0; x) = \partial h(x) + N(x)\).
+  Finally, \(\partial m(0; x, \sigma) = \nabla f(x) + \partial h(x) + N(x)\), and \(x\) is first-order stationary for~\eqref{eq:nlp}.
+  The same logic applies in the opposite direction, and establishes the equivalence.
+  \qed
+\end{proof}
+
+\begin{lemma}%
+  \label{lem:prop-p}
+  Let \Cref{asm:psi,asm:unif-prox-bounded} be satisfied, \(\nabla f(x)\) be bounded over \(\R^n\), and \(B(x) = B(x)^T \succeq 0\).
```
Comment (MohamedLaghdafHABIBOULLAH) [inline paper/rn2.tex:268]: Why do we need the boundedness of the gradient ?

**Comment by dpo** [comment]
@MohamedLaghdafHABIBOULLAH I fixed the definition of $\xi$. Again. Could you please check? It was wrong before.

**Review by MohamedLaghdafHABIBOULLAH — CHANGES_REQUESTED** [review]
In line 402, $\sigma_{max}$ should be equal to max not to min

**Comment by dpo** [comment]
> In line 402,  should be equal to max not to min

Is that a change in this PR? If not, could you submit another PR?

**Comment by MohamedLaghdafHABIBOULLAH** [comment]
> 



> > In line 402,  should be equal to max not to min
> 
> Is that a change in this PR? If not, could you submit another PR?

Alright

**Comment by dpo** [comment]
Thank you.

**Pull request merged**